What's new for .NET 9 Preview 6 (#41745)
dotnet/docs: docs/core/tools/buildcheck-rules/index.md

---
title: BuildCheck rule list
description: A complete list of BCxxxx build check errors.
ms.topic: error-reference
ms.date: 07/10/2024
f1_keywords:
- BC0101
- BC0102
---
# BuildCheck rule list

**This article applies to:** ✔️ .NET 9 SDK and later versions

> This list is a complete list of the build-check warnings that you might get from the .NET SDK.

Reviewer: "list" is used twice.

```suggestion
The following list includes all build-check warnings that you might get from the .NET SDK.
```
